src/Perlang.Common/ITypeReference.cs

```diff
@@ -79,6 +79,7 @@ ClrType switch
 
                 // Cannot use typeof(AsciiString) since Perlang.Common cannot depend on Perlang.Stdlib
                 var t when t.FullName == "Perlang.Lang.AsciiString" => true,
+                var t when t.FullName == "Perlang.Lang.Utf8String" => true,
```

Comment:
This was necessary for one of the tests to pass, and should be added here anyway.